client/src/featureform/enums.py

```diff
@@ -17,6 +17,14 @@ class ScalarType(Enum):
     BOOL = "bool"
     DATETIME = "datetime"
 
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
```

Comment:
```suggestion
          try:
            cls(value)
            return True
        except ValueError:
            return False
```

1. Doesn't depend on internal implementation of value2member map (that could change between python versions)
2. Conforms to the pythonic "easier to ask for forgiveness than permission" https://realpython.com/python-lbyl-vs-eafp/#the-easier-to-ask-forgiveness-than-permission-eafp-style

Reply:
I hate this but sure